src/react/components/DealsTable.tsx

```diff
@@ -92,7 +94,7 @@ export const DealsTable = (props: Props) => {
 				</TableCell>
 				<TableCell>{`${daysRemaining} / ${deal.timeToMaturityDays}`}</TableCell>
 				<TableCell>{`${dealStatus !== null && formatDealStatus(dealStatus)}`}</TableCell>
-				<TableCell>{userDeal && "repay"}</TableCell>
+				<TableCell>{showRepayButton && <CredixButton text="repay" />}</TableCell>
```

Comment:
If it is a button, maybe the onClick should only works on button!

Reply:
you need to be able to click on every deal that's yours, so even rows without the button are clickable.
there is no need for an onclick action here.
I used a button anyways because it's already styled and we will commence work on V2 soon anyways.